CHANGES.rst

```diff
@@ -5,6 +5,8 @@
 New Features
 ^^^^^^^^^^^^
 
+- added feature to specify extension when calling ImageFileCollection [#463]
```

Comment:
Maybe this could be specified a bit more: "Added `ext` parameter and property to ImageFileCollection to specify the extension."